### PR TITLE
chore(#42): Add SonarCloud config

### PR DESCRIPTION
**Pull request by andrablaj**
Adds a file containing the repo-level SonarCloud configuration for cht-sync.

See [the related documentation](https://docs.communityhealthtoolkit.org/contribute/code/static-analysis).

Closes https://github.com/medic/cht-sync/issues/42

### CODE REVIEW

**Comment by andrablaj** [comment]
@garethbowen Can you please enable SonarCloud for this repo by following the steps in [this documentation](https://docs.communityhealthtoolkit.org/contribute/code/static-analysis/#adding-a-new-repo-to-sonarcloud). Thanks!

**Comment by garethbowen** [comment]
@andrablaj I've done the GH <-> Sonar part. I set the number of days for the new code definition to 1 (which is what we've set it to in cht-core recently).

File: .sonarcloud.properties [hunk 1]
```diff
@@ -0,0 +1,7 @@
+sonar.sources=.
+# Can have multiple comma-separated entries
+sonar.exclusions=**/tests*/**/*
+
+sonar.tests=.
+# Can have multiple comma-separated entries
+sonar.test.inclusions=**/tests*/**/*
```
Comment (njuguna-n) [inline .sonarcloud.properties:7]: nit: Add a new line at the end.

**Review by njuguna-n — APPROVED** [review]
Left a minor comment. Thanks for adding this.

**Comment by garethbowen** [comment]
@jkuester IIRC we turned off the sonarcloud comment (above) in cht-core to reduce spam but I can't remember how we did this and for the life of me I can't find the setting. Can you remember?

**Comment by andrablaj** [comment]
@garethbowen I have just done this by Disabling Summary Content in the Pull Requests:

Screenshot 2024-02-09 at 10 27 55

**Comment by garethbowen** [comment]
:face_with_spiral_eyes: THANK YOU! I must be going blind.

**Pull request merged**